storm-server/src/main/java/org/apache/storm/scheduler/Cluster.java

```diff
@@ -889,6 +889,20 @@ public double getClusterTotalMemoryResource() {
         return sum;
     }
 
+    @Override
+    public Map<String, Double> getClusterTotalGenericResource() {
```

Comment:
Method name should include s at the end.

Reply:
Yes, addressed